src/smart-components/group/add-group/users-list.js

```diff
@@ -31,7 +33,7 @@ const createRows = (data, expanded, checkedRows = []) => {
         props: {
           data: { isActive }
         }
-      }, username, email, first_name, last_name ],
+      }, { title: <Link to={ `/users/detail/${username}` }>{username}</Link> }, email, first_name, last_name ],
```

Comment:
This will add the link in all places where users-list is used (when adding users to group etc.) I don't think we want that. I'd hide this link behind some prop.